rm-datasource/src/main/java/io/seata/rm/datasource/DataSourceProxy.java

```diff
@@ -101,6 +102,20 @@ private void init(DataSource dataSource, String resourceGroupId) {
         RootContext.setDefaultBranchType(this.getBranchType());
     }
 
+    /**
+     * test mysql adaptive type for PolarDB-X
+     *
+     * @param connection db connection
+     */
+    private void testMySQLAdaptiveType(Connection connection) {
```

Comment:
Method names need to be more standardized, generally only UT will use the test name.